src/Selenium2Library/keywords/_element.py

```diff
@@ -748,6 +758,15 @@ def _map_ascii_key_code_to_key(self, key_code):
             key = chr(key_code)
         return key
 
+    def _map_named_key_code_to_special_key(self, key_name):
+        try:
+           return getattr(Keys, key_name)
+        except:
```

Comment:
Plain `except` isn't a good idea here either. Should have `except AttributeError:` instead.